playground/frontend/lib/pages/playground/states/examples_state.dart

```diff
@@ -50,31 +50,31 @@ class ExampleState with ChangeNotifier {
   }
 
   Future<String> getExampleOutput(String id, SDK sdk) async {
-    return await _exampleRepository.getExampleOutput(
+    return _exampleRepository.getExampleOutput(
```

Comment:
Returning the future is the same as awaiting and then returning. Linter now flags this.